speedRead/server.js

```diff
@@ -0,0 +1,58 @@
+let fs = require('fs');
+let express = require('express');
+let app = express();
+let sprecord;
```

Comment:
This declaration can go within the callback from line 7 since its not being used outside of the callback.